docs/components/zeebe/technical-concepts/protocols.md

```diff
@@ -35,3 +41,7 @@ If there is no client in your target language yet, you can [build your own clien
 
 Zeebe supports [loading arbitrary gRPC server interceptors](self-managed/zeebe-deployment/zeebe-gateway/interceptors.md) to intercept incoming
 calls.
+
+:::note
+As of 8.5.0, there is no interceptor or middleware functionality for the REST part of the client yet.
```

Comment:
❓ Should we remove the indication that we'll actually add this? Underpromise, overdeliver 😄 
```suggestion
As of 8.5.0, there is no interceptor or middleware functionality for the REST part of the client.
```